src/main/java/nextstep/fp/Lambda.java

```diff
@@ -18,39 +18,28 @@ public static void printAllLambda(List<Integer> numbers) {
     }
 
     public static void runThread() {
-        new Thread(new Runnable() {
-            @Override
-            public void run() {
-                System.out.println("Hello from thread");
-            }
-        }).start();
+        new Thread(() ->   System.out.println("Hello from thread")).start();
     }
-
+    
+    // sum 추가
+    public static int sum(List<Integer> numbers, Conditional condition) {
+        return numbers.stream()
+                .filter(condition::test)
+                .reduce(0, Integer::sum);
+	}
+    
     public static int sumAll(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            total += number;
-        }
-        return total;
+        
+        return sum(numbers, n -> true);
     }
 
     public static int sumAllEven(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            if (number % 2 == 0) {
-                total += number;
-            }
-        }
-        return total;
+        
+        return sum(numbers, n -> n % 2 == 0);
     }
 
-    public static int sumAllOverThree(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            if (number > 3) {
-                total += number;
-            }
-        }
-        return total;
+    public static int sumAllOverThree(List<Integer> numbers,
+    		Conditional c) {
+        return sum(numbers, n -> n>3 );
```

Comment:
```suggestion
        return sum(numbers, n -> n > 3);
```

앞으로 미션 진행하시면서 IntelliJ 등 IDE의 코드 포맷팅 기능을 한번 적용하시면 좋을 것 같습니다!